docs/self-managed/setup/deploy/other/docker.md

```diff
@@ -49,11 +50,7 @@ For Web Modeler, we only provide multi-platform images from the following releas
 
 ### Web Modeler
 
-:::note
-Web Modeler Self-Managed is available to [enterprise customers](/reference/licenses.md#web-modeler) only.
-:::
-
-The Docker images for Web Modeler are not publicly accessible, but available to enterprise customers only from
+Docker images `<8.6.0-alpha2` for Web Modeler are not publicly accessible, but available to enterprise customers only from
```

Comment:
Same as https://github.com/camunda/camunda-docs/pull/3887/files#r1627894167: I don't think we explicitly have to mention this anymore.